src/connectors/components/OnboardingPagingDotsConnector.js

```diff
@@ -0,0 +1,17 @@
+// @flow
+
+import { connect } from 'react-redux'
+
+// import * as Constants from '../../constants/indexConstants.js'
+import type { Dispatch, State } from '../../modules/ReduxTypes'
+import PagingDots from '../../modules/UI/components/PagingDots/PagingDots.js'
+
+export const mapStateToProps = (state: State) => ({
+  totalItems: state.onBoarding.totalSlides,
```

Comment:
Are we not putting this under the `ui` area of the Reducer?

Reply:
What do you mean the ui area of the reducer?

Reply:
Because we do not want to re-render the parent component, we just want to have this component controlled by state.